src/Microsoft.Data.SqlClient/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -2,7 +2,12 @@
 	<PropertyGroup>
 		<OutputType>ClassLibrary</OutputType>
```

Comment:
Are you sure this is working and you're able to build this project? 
Output type should be _Library_ I think.

Reply:
Mistake on my part, in the properties dialog it's "Class library" with the space, but that actually corresponds to no <OutputType> in the file. This is the same as the netfx and netcore projects.

Keep in mind this project is not meant to be built yet, and it's not included in any build configuration.

Reply:
It's not just that, this project fails when building VS solution, correct? So as dev, this experience isn't great when writing code or adding tests. One has to remember to exclude this project without reason.

Main branch should be error free when building the solution too, not just via scripts.

Reply:
I encountered some hiccups with it in my IDE, yes. Building the source or test projects without the common project worked, so I assumed it was some hiccup from temp files or something. Nevertheless, I've put out a PR that fixes the issue and verified it in VS. #2907 

However, if you had concerns with this, why did you approve it in the PR that introduced it?